docs/README.md

```diff
@@ -70,6 +70,9 @@ Here follows the list of GitHub Actions topics available in the current document
     - [maven-release](#maven-release)
     - [maven-update-pom-version](#maven-update-pom-version)
     - [nexus-create-staging](#nexus-create-staging)
```

Comment:
I think the actions `nexus-create-staging`, `nexus-close-staging` and `nexus-release-staging` can be remove because they only work on Nexus 2 and we only use Nexus 3 now.